dockerfiles/services/server/Dockerfile.prod

```diff
@@ -0,0 +1,27 @@
+# Hay que generar la documentación, que consiste en arhcivos estáticos
+FROM python:3.10 AS doc-generator
+WORKDIR /workspace
+
+# Install
+RUN pip install "poetry>=1,<2"
+COPY pyproject.toml poetry.lock ./
+ENV PATH=.venv/bin:$PATH
+RUN poetry config virtualenvs.create false
+RUN poetry install
+
+# Build
+COPY docs docs
+COPY mkdocs.yml .
+COPY .git .git
```

Comment:
porque en el archivo de prod se copia el .git?

Reply:
La documentación tiene un plug-in que necesita ver los commits de git ([`mkdocs-git-revision-date-localized-plugin`](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin)).